components/lib/splitterpanel/SplitterPanel.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div ref="container" :class="cx('root')" v-bind="ptm('root', getPTOptions)" data-pc-name="splitterpanel">
+    <div ref="container" :class="cx('root')" v-bind="ptm('root', getPTOptions)" data-pc-name="splitterpanel" style="flex-grow:1;">
```

Comment:
Shouldn't this be added as a class instead of an inline style?

Reply:
It's because the problem was discovered in the unstyled mode.